6-standard-ebooks-section-patterns.rst

```diff
@@ -149,7 +149,7 @@ The :html:`<nav>` element’s top-level :html:`<ol>` element contains a list of
 :html:`<a>` descendents
 -----------------------
 
-#.	The :value:`title`, :value:`subtitle`, :value:`ordinal`, and any `related title epub semantics <https://idpf.github.io/epub-vocabs/structure/#titles>`__ are not included in ToC entries. Their usage context is only within actual heading content.
```

Comment:
This used to link to a hash, does the new version have an equivalent anchor that we can link directly to?